mimic/model/nova_objects.py

```diff
@@ -73,6 +72,32 @@ def bad_request(message, request):
     return _nova_error_message("badRequest", message, 400, request)
 
 
+def not_found(message, request):
+    """
+    Return a 404 error body associated with a Nova not found error.
+    Also sets the response code on the request.
+
+    :param str message: The message to include in the bad request body.
+    :param request: The request on which to set the response code.
+
+    :return: dictionary representing the error body.
+    """
+    return _nova_error_message("itemNotFound", message, 404, request)
```

Comment:
Can you put this back to use the symbolic `NOT_FOUND` constant?  I've noticed Mimic's code tends to drift towards magic constants. Granted, "404" is a pretty well known one, but there's no particular reason not to be consistent.  (Plus, I would really like it if one day we could stop using integers for this, integers are a bad data-type for HTTP response codes for a number of reasons, and symbolic constants make it easier to change / update that interface.)